test/framework/src/main/java/org/elasticsearch/ingest/TestIngestDocument.java

```diff
@@ -39,8 +77,10 @@ public static IngestDocument ofMetadataWithValidator(Map<String, Object> metadat
     /**
      * Create an empty ingest document for testing
      */
-    public static IngestDocument emptyIngestDocument() {
-        return new IngestDocument(new HashMap<>(), new HashMap<>());
+    public static IngestDocument emptyIngestDocumentWithDefaultVersion() {
```

Comment:
I think it would be fine to leave the original name and just add to the javadoc that the default version is added.

Reply:
Kept original, added javadoc comment.